content/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry.md

```diff
@@ -95,9 +95,9 @@ If you have a {% data variables.product.prodname_actions %} workflow that uses a
 
 2. In your {% data variables.product.prodname_actions %} workflow file, update the package url from `https://docker.pkg.github.com` to `ghcr.io`.
 
-3. Add your new {% data variables.product.prodname_container_registry %} authentication personal access token (PAT) as a GitHub Actions secret. The {% data variables.product.prodname_container_registry %} does not support using `GITHUB_TOKEN` for your PAT so you must use a different custom variable, such as `CR_PAT`. For more information, see "[Creating and storing encrypted secrets](/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)."
+3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT) as a GitHub Actions secret. {% data variables.product.prodname_container_registry %} now supports using `GITHUB_TOKEN` for your PAT. For more information, see "[Authenticating with the Github Token](/actions/configuring-and-managing-workflows/authenticating-with-the-github_token)."
```

Comment:
I think we could shorten this step to just:
```suggestion
3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT). For more information, see "[Authentication in a workflow](/actions/reference/authentication-in-a-workflow)."
```

Reply:
@hubwriter  Hi, I just did that!